pkg/monitor/cluster/podconditions.go

```diff
@@ -22,6 +22,8 @@ var podConditionsExpected = map[corev1.PodConditionType]corev1.ConditionStatus{
 
 var restartCounterThreshold int32 = 10
 
+const Preempting string = "Preempting"
```

Comment:
You can access this constant as `k8s.io/kubernetes/pkg/kubelet/events.PreemptContainer` -- you'll just have to run a `go mod vendor` after.

Reply:
Updated as suggested.